ansible_collections/arista/avd/examples/cv-pathfinder/group_vars/SITE1.yml

```
@@ -34,6 +34,7 @@ wan_router:
     uplink_switches: [site1-border1, site1-border2]
     uplink_type: p2p-vrfs # (1)!
     bgp_as: 65000
+    evpn_role: none
```

same for all - do we still need this?
```suggestion
    evpn_role: none
```